src/distribute.ts

```diff
@@ -0,0 +1,62 @@
+import { type Update, type UserFromGetMe } from "./deps.deno.ts";
+import {
+    createThread,
+    type ModuleSpecifier,
+    type Thread,
+} from "./platform.deno.ts";
+
+class UpdateThread {
```

Comment:
Confusing name. `WorkerPool` would be better.

Reply:
Thanks, I'll change it